pkg/controllers/deployment/default_deployment.go

```diff
@@ -132,3 +153,26 @@ func appendEnvVar(envVars []corev1.EnvVar, envName, envVal string) []corev1.EnvV
 	}
 	return envVars
 }
+
+func GetOAuthServerArgumentsRaw(operatorConfig *runtime.RawExtension) (map[string]interface{}, error) {
+	oauthServerObservedConfig, err := common.UnstructuredConfigFrom(
```

Comment:
retrieve the unstructured config and pass it to both `GetOAuthServerArgumentsRaw()` (why is it exported?) and `GetOAuthServerArgumentsRaw()`

Reply:
I moved the package, but forgot to unexport. Done.